Scarb.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "openzeppelin"
+version = "0.1.0"
+cairo-version = "1.1.0"
+authors = ["OpenZeppelin"]
+description = "OpenZeppelin Contracts written in Cairo for StarkNet, a decentralized ZK Rollup"
+documentation = "https://docs.openzeppelin.com/contracts-cairo"
+readme = "README.md"
+repository = "https://github.com/OpenZeppelin/cairo-contracts"
+license-file = "LICENSE"
+keywords = ["openzeppelin", "starknet", "cairo", "contracts"]
+
+[dependencies]
+starknet = ">=1.1.0"
+
+[lib]
```

Comment:
what's this section for?

Reply:
It defines a library target, so that the package can be used as Scarb dependency in other packages. 
https://docs.swmansion.com/scarb/docs/reference/targets#library